File: khalti-android/src/main/java/com/khalti/widget/KhaltiButtonInterface.java
package com.khalti.widget;


import androidx.annotation.Keep;
import android.view.View;

import com.khalti.checkOut.api.Config;

@Keep
public interface KhaltiButtonInterface {
    void setText(String buttonText);

    void setCheckOutConfig(Config config);

    void setCustomView(View view);

    void setButtonStyle(ButtonStyle style);

    void showCheckOut();

    void showCheckOut(Config config);

    void destroyCheckOut();
}

File: khalti-android/src/main/java/com/khalti/widget/KhaltiButtonInterface.kt
package com.khalti.widget


import androidx.annotation.Keep
import android.view.View

import com.khalti.checkout.helper.Config

@Keep
interface KhaltiButtonInterface {
    fun setText(buttonText: String)

    fun setCheckOutConfig(config: Config)

    fun setCustomView(view: View)

    fun setButtonStyle(style: ButtonStyle)
}
